beater/handlers.go

```diff
@@ -102,15 +107,64 @@ func frontendSwitchHandler(feSwitch bool, h http.Handler) http.Handler {
 		if feSwitch {
 			h.ServeHTTP(w, r)
 		} else {
-			sendStatus(w, r, 403, errForbidden)
+			sendStatus(w, r, http.StatusForbidden, errForbidden)
+		}
+	})
+}
+
+func rateLimitHandler(rateLimit int, h http.Handler) http.Handler {
```

Comment:
Can we call it something like `rateLimitPerIpHandler` as we'll probably need a more generic one later on for backend?